api/src/main/java/com/github/retrooper/packetevents/util/MojangAPIUtil.java

```diff
@@ -76,6 +77,51 @@ public static List<TextureProperty> requestPlayerTextureProperties(UUID uuid) {
         return null;
     }
 
+    /**
+     * Here you can get a full User Profile with Texture in only one HTTP Request
+     */
+    public static UserProfile createUserProfile(UUID uuid) {
+        String uuidStr = UUIDUtil.toStringWithoutDashes(uuid);
```

Comment:
uuidStr doesn't really match what it does, you remove the dash and don't just convert it to a string

Reply:
?

Reply:
My understanding is that a UUID string is nothing more than, well, a UUID object converted to a string.
But if you remove the default dashes, you could probably give it a different name, since you are "_changing_" the "_default behavior_"

Reply:
Okay I just copied from the Get Textures Method above